### ISSUE TITLE
[Contrast and accessibility] CSS colors to meet minimum W3C web accessibility requirements on color and contrast in a standardised style module

### ISSUE THREAD

**Issue by waddie96** [SemanticMediaWiki/SemanticMediaWiki#6195]
[Wikimedia Codex design system](https://doc.wikimedia.org/codex/latest) a great standard for us to incorporate. Especially I've found the colors need attention [Codex colors](https://doc.wikimedia.org/codex/latest/style-guide/colors.html).

I did some color contrast checking and we failed, a lot.

> Tabs at top of main page: Foreground: `#5F9DC8` Background: `#FFFFFF` Contrast ratio: **2.94:1**
> 1.4.3 Contrast (Minimum) (AA) **Fail** for large and regular text
> 1.4.6 Contrast (Enhanced) (AAA) **Fail** for large and regular text
> 1.4.11 Non-text Contrast (AA) **Fail** for UI components and graphical objects

Many more, but I won't bore. See WCAG 2.0 guidelines.

<img width="478" height="712" alt="Image" src="https://github.com/user-attachments/assets/11316c67-fa67-417e-92e9-e7433e074500" />

**Also**, the ridiculously small ? help icon and ... ellipsis
